scripts/spark_eval/copy_exome_to_hdfs.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+# Download all required data for exomes and store in HDFS.
+
+TARGET_DIR=${1:-exome_spark_eval}
+
+hadoop fs -stat $TARGET_DIR > /dev/null 2>&1
+if [ $? -eq 0 ]; then
+  echo "$TARGET_DIR already exists. Delete it and try again."
+  exit 1
+fi
+
+set -e
+set -x
+
+# Create data directory in HDFS
+hadoop fs -mkdir -p $TARGET_DIR
+
+# Download exome BAM (ftp://ftp-trace.ncbi.nih.gov/1000genomes/ftp/technical/working/20101201_cg_NA12878/)
+gsutil cp gs://broad-spark-eval-test-data/data/NA12878.ga2.exome.maq.raw.bam - | hadoop fs -put - $TARGET_DIR/NA12878.ga2.exome.maq.raw.bam
```

Comment:
Should we use the -m command for parallel download?  That seems to speed things up for me.

Reply:
Oh, nm, I didn't read far enough to see that it it was being streamed into hadoop....